mjcf_to_sdformat/mjcf_to_sdformat/converters/link.py

```diff
@@ -90,7 +91,24 @@ def mjcf_geom_to_sdf(body):
                                     inertial_euler.pitch(),
                                     inertial_euler.yaw()))
         link.set_inertial(inertial)
-
+    else:
+        try:
+            body_inertia = physics.named.model.body_inertia[body.name]
+            inertial = Inertiald(
+                MassMatrix3d(physics.named.model.body_mass[body.name],
+                             Vector3d(body_inertia[0],
+                                      body_inertia[1],
+                                      body_inertia[2]),
+                             Vector3d(0, 0, 0)),
+                Pose3d(physics.named.model.body_ipos[body.name][0],
+                       physics.named.model.body_ipos[body.name][1],
+                       physics.named.model.body_ipos[body.name][2],
+                       0,
```

Comment:
I think we should also include the orientation from `body_iquat`

Reply:
https://github.com/gazebosim/gz-mujoco/pull/48/commits/1134f6a69830fde4dc29bab9ff715b105dc4656e